noir-projects/noir-contracts/contracts/claim_contract/Nargo.toml

```diff
@@ -8,3 +8,4 @@ type = "contract"
 aztec = { path = "../../../aztec-nr/aztec" }
 uint_note = { path = "../../../aztec-nr/uint-note" }
 token = { path = "../token_contract" }
+crowdfunding = { path = "../crowdfunding_contract" }
```

Comment:
This change is related to [this comment](https://github.com/AztecProtocol/aztec-packages/pull/11904#discussion_r1954122823).